advanced/index.md

```diff
@@ -144,7 +144,7 @@ Cutting and creating releases on the command line and on GitHub.
     * [Git-Flow: A Successful Branching Model](http://nvie.com/posts/a-successful-git-branching-model/)
     * [Git-Flow Source](https://github.com/nvie/gitflow)
     * Too many levels?
-    * GH prefers Simplest thing that works.
+    * GH prefers "Simplest thing that works".
```

Comment:
Might be good to either move this period inside the quote or drop it off entirely to remain consistent with other lists.